gms/gossiper.cc

```diff
@@ -2173,6 +2173,11 @@ future<> gossiper::wait_for_range_setup() {
     return wait_for_gossip(ring_delay);
 }
 
+void gossiper::set_node_to_be_replaced(gms::inet_address node) {
+    logger.info("Set node to be replaced = {}", node);
+    _node_to_be_replaced = node;
```

Comment:
Where is this used? github gui is so bad I cannot find it.

Reply:
It is used in storage_service: Update tokens and replace address for replace operation.

Reply:
I see it is only set there, never used. If it is indeed used anywhare sending a proper patchset to a mailing list would made it much easier for me to find.

Reply:
The actual user is in https://github.com/scylladb/scylla/pull/5154. I got some of the patches from that PR because they all are handling the replacing operations.  I do not want this PR to depend on 5154.

Reply:
First of all it is good that I am not going crazy by not seeing its use here. But why not introduce it in the series it is used in than?

Reply:
I can get rid of this patch from this series to reduce the noise.